### PR TITLE
Add info for packaging the project

### PR DESCRIPTION
**Pull request by microbecode**
- Add info for npm package manager
- Bump hardhat version - fixes unit tests deployments somewhat

### CODE REVIEW

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>013de5a</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-8n685pt6i-equilibriumco.vercel.app'>https://starknet-multisig-8n685pt6i-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/FFEVjnNW3k2P12XPj1U8dEi5AGNW'>https://vercel.com/equilibriumco/starknet-multisig/FFEVjnNW3k2P12XPj1U8dEi5AGNW</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2858944095)

File: contracts/package.json [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+  "version": "0.2.1",
+  "author": "Equilibrium (www.equilibrium.co)",
   "license": "MIT",
   "private": false,
+  "homepage": "https://github.com/eqlabs/starknet-multisig",
+  "keywords": [
+    "multisig",
+    "starknet"
+  ],
+  "description": "A multi-signature contract for StarkNet L2",
```
Comment (JaniAnttonen) [inline contracts/package.json:12]: I'd drop the L2 mention here.

**Review by JaniAnttonen — CHANGES_REQUESTED** [review]
Small comment about wording in the description, otherwise all good

File: contracts/package.json (hunk 1 above)
Comment (microbecode) [reply contracts/package.json]: fix'd!

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>462ecfb</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-cqwv44gcb-equilibriumco.vercel.app'>https://starknet-multisig-cqwv44gcb-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/D4tKF3n9mMoEHSaiFY3DcBxk4Rjr'>https://vercel.com/equilibriumco/starknet-multisig/D4tKF3n9mMoEHSaiFY3DcBxk4Rjr</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2859582564)

**Review by JaniAnttonen — APPROVED** [review]

**Pull request merged**